tox.ini

```diff
@@ -43,6 +43,7 @@ deps =
     Sphinx
 
 [pep8]
+ignore = E123,E266
```

Comment:
Disabling E123 is needed in the PR because once you disable *one* error (E266), the out-of-the-box defaults are no longer in effect:  we have a bunch of places where E123 then triggers.

The kind of comments you propose make the handling a bunch more error-prone than the doubled `##'.